### ISSUE TITLE
Typo in provider info, spis etc  : "twelvemkonkeys"

### ISSUE THREAD

**Issue by balamuru** [haraldk/TwelveMonkeys#258]
These typos affect the instantiation of writers from the associated readers. There are several instantances of the typo (search URL :  https://github.com/haraldk/TwelveMonkeys/search?utf8=%E2%9C%93&q=twelvemkonkeys&type=Code )

eg.

``````java
protected TIFFProviderInfo() {
        super(
                TIFFProviderInfo.class,
                new String[] {"tiff", "TIFF", "tif", "TIF"},
                new String[] {"tif", "tiff"},
                new String[] {
                        "image/tiff", "image/x-tiff"
                },
                "com.twelvemkonkeys.imageio.plugins.tiff.TIFFImageReader", ================> typo
                new String[] {"com.twelvemonkeys.imageio.plugins.tiff.TIFFImageReaderSpi"},
                "com.twelvemonkeys.imageio.plugins.tiff.TIFFImageWriter",
                new String[] {"com.twelvemkonkeys.imageio.plugins.tif.TIFFImageWriterSpi"}, ========> typo
                false, TIFFMedataFormat.SUN_NATIVE_STREAM_METADATA_FORMAT_NAME, "TODO", null, null,
                true, TIFFMedataFormat.SUN_NATIVE_IMAGE_METADATA_FORMAT_NAME, "com.twelvemonkeys.imageio.plugins.tiff.TIFFImageMetadata", null, null
        );
    }
```````

**Comment by haraldk** [comment]
Hi Vinay,

Thank you for reporting! 
Good catch!  This is actually quite embarrassing.. ;-)

I'll create a patch right away!

-- 
Harald K

**Comment by haraldk** [comment]
Fixed in 3d68b61.

-- 
Harald K

**Comment by balamuru** [comment]
Thanks for the timely fix. When will the changes propagate to the maven repo?

**Comment by haraldk** [comment]
I might release a bugfix release at some time, but there are quite a few open bugs I’d like to fix before 3.3 release. As this is a hobby project, I make no guarantees about release dates. :-)

-- 
Harald K
